plugin/httptrace/server.go

```diff
@@ -0,0 +1,156 @@
+package httptrace
+
+import (
+	"io"
+	"net/http"
+
+	"go.opentelemetry.io/api/core"
+	"go.opentelemetry.io/api/tag"
+	"go.opentelemetry.io/api/trace"
+)
+
+var _ http.Handler = &httpHandler{}
+
+// HTTPHandler provides http middleware that corresponds to the http.Handler interface
+type httpHandler struct {
+	operation string
+	handler   http.Handler
+	tracer    trace.Tracer
+}
+
+// NewHandler wraps the passed handler in an span named after the operation and
+// with provided options, and functions as http middleware. The span is tagged
+// with:
+//   * "read_bytes" - if anything was read from the request body
+//   * "wrote_bytes" - if anything was written to the response writer
+//   * "http_status" - the http status, if set
```

Comment:
ochttp has these [attributes](https://github.com/census-instrumentation/opencensus-go/blob/fa651b05963cfb6060755dc887e7d156ba66e792/plugin/ochttp/trace.go#L33-L40). These attributes were as per the[ HTTP plugin ](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md)spec. Eventually this spec (or some modified version of it) will be part of Otel-Spec. 

Perhaps add a TODO to add these attributes.